release/_bom.sh

```diff
@@ -319,6 +319,12 @@ function generate_poms {
 }
 
 function generate_poms_from_scratch {
+	mkdir -p "${_BUILD_DIR}/boms"
```

Comment:
@pgydev, what happens if this folder already exists? Shouldn't we remove it at first and then create it?

Reply:
@natocesarrego, because of the -p flag the `mkdir` won't create the directory if it's already exist.